cdap-ui/app/directives/log-viewer/log-viewer.js

```diff
@@ -268,6 +271,7 @@ function LogViewerController ($scope, $window, LogViewerStore, myLogsApi, LOGVIE
       //otherwise, it does not have stack trace but has been selected
       vm.displayData[index].selected = !vm.displayData[index].selected;
     }
+    checkForScrollbar();
```

Comment:
Talked to @tonybach. This is a problem in the existing log viewer too (table headers won't align with table columns if scroll bar appears).